docs/posts/overview-of-ballerina-0.982.0.md

```diff
@@ -0,0 +1,361 @@
+---
+title: Overview of Ballerina 0.982.0
+author: Maryam Ziyad Mohamed
+date: 01 October 2018
+status: Published
+abstract: Introduction to New Features and Changes introduced with Ballerina 0.982.0
+socialmediaimage: <img src="http://getdrawings.com/img/ballerina-silhouette-1.jpg" />
+---
+
+# Improvements and New Features
+The Ballerina 0.982.0 release includes new features, improvements and bug fixes including changes to the `main` 
+function, introduction of the new `channel` type, improvements to records and objects and support for HTTP 1.1 
+piplelining.
+
+An initial version of the Ballerina Native feature that allows simple Ballerina programs with the `main` function to be 
+compiled into native executables, is also introduced with this release.
+
+## Language & Runtime
+
+### Introduction of the `channel` Type
```

Comment:
I'm not sure how backticks in headings will render when this is generated to the website. I think it's better to avoid if possible. 

@madhuramendis WDYT?